make/out.mk

```diff
@@ -0,0 +1,9 @@
+ifndef OUT_MK
```

Comment:
Could we remove make/.out? Don't think we need this :)

Reply:
It's used by `check-go-fmt`. It creates a file inside `out` with the list of the found errors.